index.html

```diff
@@ -477,7 +477,7 @@
   // Fill version selector.
   let select = $("#select_version");
   let allVersions = [... (new Set(versions.map(v => parseInt(v[1])))).values()]
-                      .sort()
+                      .sort((a,b)=>a-b)
```

Comment:
That's not according to the code style. In https://github.com/mozilla/usecounters/blob/gh-pages/index.html#L354 there are spaces between argument definitions and also between `=>`
There are also spaces between math operators and operands:
https://github.com/mozilla/usecounters/blob/gh-pages/index.html#L312
https://github.com/mozilla/usecounters/blob/gh-pages/index.html#L391

Reply:
thanks will make the necessary changes